README.md

```diff
@@ -1,129 +1,115 @@
 # Buf for Visual Studio Code
 
-The [VS Code Buf extension](https://marketplace.visualstudio.com/items?itemName=bufbuild.vscode-buf) provides rich support for [Protocol Buffers](https://protobuf.dev/) using the [Buf](https://buf.build/) development environment.
+The [VS Code Buf extension][vs-code-marketplace] provides rich support for [Protocol Buffers][protobuf] using the [Buf][buf] development environment.
```

Comment:
Might suggest being a bit more direct with the problem we're solving. Something like...

> The [VS Code Buf extension][vs-code-marketplace] helps you work with [Protocol Buffers][protobuf] files in a much more intuitive way, adding smart syntax highlighting, navigation, formatting, and Buf command support.